presto-orc/src/test/java/com/facebook/presto/orc/TestTupleDomainOrcPredicate.java

```diff
@@ -172,6 +174,31 @@ public void testString()
         assertEquals(getDomain(VARCHAR, 10, stringColumnStats(5L, "apple", null)), create(ValueSet.ofRanges(greaterThanOrEqual(VARCHAR, utf8Slice("apple"))), true));
     }
 
+    @Test
+    public void testChar()
+        throws Exception
+    {
+        assertEquals(getDomain(CHAR, 0, null), none(CHAR));
+        assertEquals(getDomain(CHAR, 10, null), all(CHAR));
+
+        assertEquals(getDomain(CHAR, 0, stringColumnStats(null, null, null)), none(CHAR));
```

Comment:
Add tests where statistics have differant number of spaces (so total number of characters does not match length of CHAR(x) type)